docs/grammar.md

```diff
@@ -111,13 +110,7 @@ rule_name[return_type]:
     | first_alt1 first_alt2 { first_alt1 }
     | second_alt1 second_alt2 { second_alt1 }
 ```
-If the action is omitted and C code is being generated, then there
-are two different possibilities:
-1.  If there's a single name in the alternative, this gets returned.
-2.  If not, a dummy name object gets returned.
-
-If Python code is being generated, then a list with all the parsed
-expressions gets returned.
+If the action is omitted, a list with all the parsed expressions gets returned.
```

Comment:
This should probably be changed after #14, right?

Reply:
Indeed

Reply:
Addressed.

Reply:
Thanks.